### DISCUSSION TITLE
Accessibility Advantage over Axe

### DISCUSSION THREAD

**Discussion by Leapschel** [GoogleChrome/lighthouse#14524] [General]
Hi, I think you are using axe core for accessibility. However I was wondering if there are any additional rules you consider or test against? Or is there an advantage to using your accessibility report to using axes browser extension?

thanks 😊

**Comment by brendankenny** [comment]
Yes, Lighthouse uses https://github.com/dequelabs/axe-core for the Accessibility category, with no additional rules added.

Only advantage is its in a tool you might otherwise be using, otherwise the [axe extension](https://www.deque.com/axe/browser-extensions/) has all the checks Lighthouse uses and more.
